### ISSUE TITLE
As a user I want improvements to the graph tooltips

### ISSUE THREAD

**Issue by avlisodraude** [beeldengeluid/labo-components#318]
As a user I want improvements to the graph tooltips.

**Where?**
In the Comparison, Single Search, Collection inspector tools.

**Comparison tool:** Number separator and layout issues. Data too cram (see screenshot).
![Screenshot 2019-03-18 at 18.16.15.png](https://images.zenhubusercontent.com/5a1e745d8a75884b90892ac4/0ed7f0d4-0ee3-453c-9ad2-6f96012955d9)

**Collection inspector:** Number separator
![Screenshot 2019-03-18 at 18.18.24.png](https://images.zenhubusercontent.com/5a1e745d8a75884b90892ac4/daeae1c8-284e-4236-87d7-55419441eac1) 

**Single Search:** Number separator for Histogram and Line chart
![Screenshot 2019-03-18 at 18.21.06.png](https://images.zenhubusercontent.com/5a1e745d8a75884b90892ac4/0eaa28b7-ed1d-45d5-992a-1b121116ce56)
![Screenshot 2019-03-18 at 18.21.15.png](https://images.zenhubusercontent.com/5a1e745d8a75884b90892ac4/cc40b3a0-4a73-4bf2-9752-56a99853e924)

**Acceptance Criteria**
- [ ] Add separator for numbers in query comparison tooltips
- [ ] Style comparison tooltips to avoid issues with cram data
- [ ] In collection inspector add separator to numbers in tooltip
- [ ] In Single search add separator to numbers in tooltip

**Comment by avlisodraude** [comment]
@lilimelgar @jblom 

Issue for labo-components project done in branch:
_issue/#318-enhancement-to-graph-tooltips_

**What's been done?**
In comparison tool a separator for numbers has been added in tooltips.
In comparison tool styling for tooltips data.
In collection inspector a separator for numbers has been added in tooltip.
In Single search a separator for numbers in tooltip has been added.
For every graph modified its corresponding Y axis has been modified accordingly.

**Screencast**
https://drive.google.com/open?id=1w3AlhdIYUsVPpSs9NWC_P-_xBsn_3WzR

**Comment by jblom** [comment]
@avlisodraude alright I tested it and went a little bit overboard by refactoring the collection inspector recipe (and related components). Related to this issue I also fixed:

- number formatting of somenumber.toFixed(2) -> this resulted in always point notation rather than using the locale of the user
- the legend of the CollectionInspectorLineChart (now renamed to MetadataCompletenessChart) title displayed the selected date field rather than the analysis field (image is how it is now, showing the analysis field correctly): 
![Schermafbeelding 2019-03-20 om 16 42 19](https://user-images.githubusercontent.com/917310/54698223-3b2f1c00-4b2f-11e9-97eb-e60848725aa3.png)

Ok I merged it with master already, because I need to move on to your PR's related to the single search.

@lilimelgar should still test though (using master). Also let me know when you're ready, so I can add a test plan
